.github/workflows/data-checks.yaml

```
@@ -0,0 +1,47 @@
+name: 'Data checks'
+
+on:
+  schedule:
```

One final suggestion, for all Workflows that use a cron schedule, I generally add a `workflow_dispatch:` trigger. This will enable the action to run on demand via the Actions tab. Useful for debugging, testing and ad-hoc runs.

```suggestion
  workflow_dispatch:
  schedule:
```